examples/traditional-vue-app/README.md

```diff
@@ -0,0 +1,18 @@
+# vue-hcaptcha demo app
+
+Demo app to showcase usage of the vue web component.
+
+## Project setup
+```
+npm install
+```
+It uses latest version of `@hcaptcha/vue-hcaptcha`.
+
+When developing, make sure to link the local version of the component. Run followings:
+* on root: `npm link`
```

Comment:
Are you installing the npm hosted version and showing that vs running the local repo one?

Reply:
Indeed. The demo is about showing end usage, but can also be used when developing by doing a local npm link.

Reply:
Isn't the point of pulling / forking this repository to work on it? The first thing I would do is run npm start thinking I can spin up code to work with.

Reply:
Sure thing. Let's favor development over end usage example. 

Now it is possible to do `npm run serve` in one of the examples and a live-reload webpage is served which reflect changes of the `vue-hcaptcha` web component in real-time.